lib/features/manage_account/domain/model/create_new_identity_request.dart

```diff
@@ -2,14 +2,23 @@
 import 'package:equatable/equatable.dart';
 import 'package:jmap_dart_client/jmap/core/id.dart';
 import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:model/identity/identity_request_dto.dart';
 
 class CreateNewIdentityRequest with EquatableMixin {
 
   final Identity newIdentity;
   final Id creationId;
+  final List<IdentityRequestDto>? updateRequest;
+  final List<IdentityId>? updateIdentityIds;
```

Comment:
what use for?

Reply:
it's used for Create new default identity request, i think i should create new class extends this class for create default identity